queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND  escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name NOT IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT name FROM animals WHERE neutered = TRUE;
+SELECT name FROM animals WHERE name NOT IN ('Gabumon');
+SELECT name FROM animals WHERE weight_kg >= 10.5 AND weight_kg <= 17.3; ;
```

Comment:
- [ ] Kind remainder I know that might be mistyped but to be sure we are supposed to query from 10.4kg to 17.3kg, Also I noticed we have extra semi-colon 😇

Reply:
Asked change implemented